### DISCUSSION TITLE
Welcome to PlayBook Discussions!

### DISCUSSION THREAD

**Discussion by Goodwy** [Goodwy/PlayBooks#29] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by intrepidclass** [comment]
Hi I just started using this app. After looking at a few audio book players this one seemed the best and prettiest. And I like that it's open source.
I'm happy to give feedback, also maybe I can make some time to do some development.
